bitnami/redis/values.yaml

```diff
@@ -138,6 +138,59 @@ sentinel:
     labels: {}
     loadBalancerIP:
 
+  ## Prometheus Exporter / Metrics for Redis Sentinel
+  ##
+  metrics:
+    enabled: false
+
+    image:
+      registry: docker.io
+      repository: leominov/redis_sentinel_exporter
+      tag: 1.7.1
+      pullPolicy: IfNotPresent
+      ## Optionally specify an array of imagePullSecrets.
+      ## Secrets must be manually created in the namespace.
+      ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+      ##
+      # pullSecrets:
+      #   - myRegistryKeySecretName
+
+    ## Sentinel metrics exporter resource requests and limits
+    ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    ##
+    # resources: {}
+
+    ## Extra arguments for Sentinel metrics exporter, for example:
+    ## extraArgs:
+    ##   check-keys: myKey,myOtherKey
+    # extraArgs: {}
+
+    # Enable this if you're using https://github.com/coreos/prometheus-operator
+    serviceMonitor:
+      enabled: false
+      ## Specify a namespace if needed
+      # namespace: monitoring
+      # fallback to the prometheus default unless specified
+      # interval: 10s
+      ## Defaults to what's used if you follow CoreOS [Prometheus Install Instructions](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#tldr)
+      ## [Prometheus Selector Label](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#prometheus-operator-1)
+      ## [Kube Prometheus Selector Label](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#exporters)
+      selector:
+        prometheus: kube-prometheus
+
+    service:
+      type: ClusterIP
+
+      ## External traffic policy (when service type is LoadBalancer)
+      ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+      externalTrafficPolicy: Cluster
+
+      ## Use serviceLoadBalancerIP to request a specific static IP,
+      ## otherwise leave blank
+      # loadBalancerIP:
+      annotations: {}
+      labels: {}
```

Comment:
You need to update **values-production.yaml** too

Reply:
Done, should the `production-sentinel-values.yaml` file also be adjusted. I'm not sure if the exporter should be enabled for the CI pipeline?

Reply:
I think so, the more combinations we test in the CI, the better.

Reply:
As of today, `values-production.yaml` is no longer included so I'm marking this conversation as resolved.